src/GraphQL/GraphQLAttribute.cs

```diff
@@ -69,7 +69,8 @@ public virtual void Modify(ArgumentInformation argumentInformation)
             var typedMethod = _typedModifyMethods.GetOrAdd(
                 argumentInformation.ParameterInfo.ParameterType,
                 type => _modifyMethod.MakeGenericMethod(type));
-            typedMethod.Invoke(this, new object[] { argumentInformation });
```

Comment:
Why did you change implementation? Faster? Safer?

Reply:
Exceptions were wrapped with TargetInvokeException previously

Reply:
I also added a test which exercises both bits of changed code and verifies that exceptions bubble without being wrapped